includes/BucketDatabase.php

```diff
@@ -36,6 +36,7 @@ public static function getDB(): IMaintainableDatabase {
 
 		self::$db = MediaWikiServices::getInstance()->getDatabaseFactory()->create( $mainDB->getType(), $params );
 
+		// TODO: figure out new min version
 		// MySQL 8.0.17 or higher is required for the implementation of repeated fields.
 		if ( self::$db->getType() !== 'mysql' || version_compare( self::$db->getServerVersion(), '8.0.17', '<' ) ) {
 			throw new ConfigException( 'Bucket requires MySQL 8.0.17 or higher' );
```

Comment:
Maybe we remove this check? If someone reports it crashing and burning on a particular version then we can revisit this and/or add something to the README, but it seems to me like the compatibility expands a decent amount just by switching the repeated fields implementation.